ci-operator/jobs/openshift-knative/serverless-operator/openshift-knative-serverless-operator-master-periodics.yaml

```diff
@@ -79,7 +79,7 @@ periodics:
         secretName: result-aggregator
 - agent: kubernetes
```

Comment:
Is this job then still needed ? Looks like a duplication of openshift-knative-serverless-operator-master__4.3.yaml

Reply:
It's needed. Yes. The file `openshift-knative-serverless-operator-master__4.3.yaml` is a complement of this. It includes images that should be built etc.

Reply:
And this stuff on line 80 which you referenced is actually generated from that file.